src/components/ReservationListCard/ReservationListCard.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled, { css } from 'styled-components/macro';
+
+const ReservationListCard = ({
+  id,
+  name,
+  price,
+  src,
+  origin,
+  destination,
+  departure_time,
+  arrival_time,
+  duration,
+  flight_name,
+}) => {
+  const navigate = useNavigate();
+  const goToPayment = () => {
+    navigate(`/payment/${id}`);
+  };
+
+  const [time, minute] = duration.split(':');
+
+  const convertTime = time => {
+    return (time =
+      time[0] === '0' ? `${time[1]}` + '시간' : `${time}` + '시간');
+  };
```

Comment:
리터럴이랑 + 랑 같이 써서 생긴 리뷰 같아요!
그리고, string을 num으로 바꿔주면, 0이 자동으로 없어지기 때문에 아래와 같이 convertTime은 좀 더 간결하게 표현할 수 있을 것 같아요!
```suggestion
  const convertTime = time => {
    return `${Number(time)}시간`;
  };
```